cmd/rpcfuzz/rpcfuzz.go

```diff
@@ -529,6 +528,23 @@ func setupTests(ctx context.Context, rpcClient *rpc.Client) {
 		Flags:     FlagRequiresUnlock,
 	})
 
+	// $ curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_sendTransaction","params":[],"id":1}' http://localhost:8545
+	// {"jsonrpc":"2.0","id":1,"error":{"code":-32602,"message":"missing value for required argument 0"}}
+	allTests = append(allTests, &RPCTestGeneric{
+		Name:      "RPCTestEthSendTransactionEmpty",
+		Method:    "eth_sendTransaction",
+		Args:      []interface{}{},
```

Comment:
Should I remove these lines or keep it just to make it easier to read?

Reply:
is it better to have a section in the README (or a README for rpcfuzz) that keeps these lines to explain what the test is for?

Reply:
i personally think keep it, it makes it clear that it's passing a [] to the JSON RPC.